app/Services/ModuleService.php

```diff
@@ -97,11 +95,11 @@ public function createModule($array)
 
         $root_files = scandir($temp);
 
-        if (!in_array( 'module.json', $root_files)) {
+        if (!in_array('module.json', $root_files)) {
             $temp .= '/'.$root_files[2];
         }
 
-        foreach(glob($temp.'/*.json') as $file) {
+        foreach (glob($temp.'/*.json') as $file) {
```

Comment:
It's better to check using `file_exists()`

Reply:
Done!